Source/Core/Core/PowerPC/JitCommon/JitBase.h

```diff
@@ -104,6 +105,7 @@ class JitBase : public CPUCoreBase
 		JitBlock *curBlock;
 
 		std::unordered_set<u32> fifoWriteAddresses;
+		std::unordered_set<u32> pairedQuantizeAddresses;
```

Comment:
If we're going to continue adding cases here, might be better to make this an array of unordered_sets to reduce the amount of boilerplate.